website/src/graphql/dailp/queries.graphql

```diff
@@ -259,6 +259,8 @@ query DocSlice(
   $morphemeSystem: CherokeeOrthography!
 ) {
   document(slug: $slug) {
+    title
+    url
```

Comment:
I added "url" here thinking it was something else, and need to remove it, but I keep getting errors when trying to do so even though I've rerun dev-generate-types. VSCode keeps telling me to run dev-check, which I do and get no issues, but then at the very end it tells me to run dev-check again. What should I do?

Reply:
Are you still having this issue?